pydicer/cli/run.py

```diff
@@ -28,7 +27,7 @@
 }
 
 PIPELINE_TOOLS = {
-    "e2e": run_test,
+    # "e2e": run_test, TODO This broke due to some changes. Either we need to fix or remove.
```

Comment:
Well I deleted the run_test function since that's not really the way its intended to run anymore... If we want the end-to-end test then we can easily put it back it. It's basically equivalent to just pulling the test data and the calling the run_pipeline function :)